### ISSUE TITLE
app.get('token') doesn't work after a browser refresh.

### ISSUE THREAD

**Issue by delay** [feathersjs/feathers-authentication#303]
On the client, I am storing my authentication token to local storage and using websockets on the client to connect.  If I use app.get('token') it works fine until I refresh the browser then it returns undefined.  If I get the jwt token directly with localStorage.getItem() it works fine even after a browser refresh.

Doesn't work:
```
// doesn't work after you refresh browser
function requireAuth (to, from, next) {
  console.log('via feathers method: '+services.app.get('token'));
  console.log('via localstorage method: '+localStorage.getItem("feathers-jwt"));
if (services.app.get('token') === null) {
    next({
      path: '/login',
      query: { redirect: to.fullPath }
    })
  } else {
    next()
  }
}
```

Works:
```
//This works fine after browser refresh
function requireAuth (to, from, next) {
  console.log('via feathers method: '+services.app.get('token'));
  console.log('via localstorage method: '+localStorage.getItem("feathers-jwt"));
  if (localStorage.getItem("feathers-jwt") === null) {
    next({
      path: '/login',
      query: { redirect: to.fullPath }
    })
  } else {
    next()
  }
}
```

Is this a bug or do I not have a proper understanding of how app.get('token') should work?

**Comment by daffl** [comment]
`token` and `user` will only be set after a successful call to `app.authenticate().then()` (with no parameters) which will try to authenticate the token currently in localStorage.

**Comment by delay** [comment]
Ok I understand that you can't really know if a token is valid or not until you run `app.authenticate().then()` but once I have a token (stored to local storage) supplied by `app.authenticate().then()` if I try to use `app.get('token')` it works fine until I hit refresh on the browser, then it becomes `undefined` however I still have a valid token in local storage.  

I would think `app.get('token')` should still return a token even after refreshing a browser provided the token still exists in local storage.

**Comment by marshallswain** [comment]
@delay, this is already fixed in the `0.8.0` version, which is in beta.  It's still a bit rough around the edges, so I recommend steering clear of it unless you have some time to help debug.

**Comment by delay** [comment]
Ahh ok...  glad you already got it fixed for the next version.
